includes/classes/ExternalConnections/WordPressExternalConnection.php

```diff
@@ -706,7 +706,28 @@ public function push( $post, $args = array() ) {
 		);
 
 		// Action documented in includes/classes/InternalConnections/NetworkSiteConnection.php.
-		do_action( 'dt_push_post', $response, $post_body, $type_url, $post_id, $args, $this );
+		do_action_deprecated(
+			'dt_push_post',
+			array( $response, $post_body, $type_url, $post_id, $args, $this ),
+			'2.0.0',
+			'dt_push_network_post|dt_push_external_post',
+			esc_html__( 'The dt_push_post action has been deprecated. Please use dt_push_network_post or dt_push_external_post instead.', 'distributor' )
```

Comment:
```suggestion
			'dt_push_network_post|dt_push_external_post'
```

With the message the notice will double up the info. 

> Hook dt_push_post is <strong>deprecated</strong> since version 2.0.0! Use dt_push_network_post|dt_push_external_post instead. The dt_push_post action has been deprecated. Please use dt_push_network_post or dt_push_external_post instead.